tests/pyincore_incubator/analyses/electricpowernetworkrecovery/test_electricpowernetworkrecovery.ipynb

```diff
@@ -0,0 +1,478 @@
+{
```

Comment:
I think we need to have a .py file rather than a note book here. You can use Chris's PR for reference https://github.com/IN-CORE/pyincore-incubator/blob/4-chris-example-building-damage-analysis/tests/pyincore_incubator/analyses/cohortbuildingdamage/test_cohortbuildingdamage.py